frontiers/the_htm_spatial_pooler_a_neocortical_algorithm_for_online_sparse_distributed_coding/README.md

```diff
@@ -0,0 +1,31 @@
+Spatial Pooler Research
+=======================
+
+This repository contains research scripts for the HTM spatial pooler (SP).
```

Comment:
This repository contains research scripts for the following paper:

The HTM Spatial Pooler—A Neocortical Algorithm for Online Sparse Distributed Coding
https://www.frontiersin.org/articles/10.3389/fncom.2017.00111